spec/rails_helper.rb

```diff
@@ -7,6 +7,13 @@
 require 'rspec/rails'
 # Add additional requires below this line. Rails is not loaded until this point!
 
+require 'simplecov'
+SimpleCov.start
+if ENV['CI']=='true'
```

Comment:
Surrounding space missing for operator `==`.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.